src/Provider/Node/SystemNodeProvider.php

```diff
@@ -73,4 +78,35 @@ protected function getIfconfig()
 
         return ob_get_clean();
     }
+
+    /**
+     * Returns mac address from the first system interface via the sysfs interface
+     *
+     * @return string|bool
+     */
+    protected function getsysfs()
+    {
+        $mac = false;
+        if (strtoupper(php_uname('s')) === "LINUX") {
+            // get all the macadresses of all systems
+            $macs = array_map(
+                'file_get_contents',
+                glob('/sys/class/net/*/address', GLOB_NOSORT)
```

Comment:
It's been a while since this was written and merged, but this will fail on some systems and needs error handling.  Here's what I'm seeing:

> file_get_contents(/sys/class/net/eth0/address): failed to open stream: Permission denied